### DISCUSSION TITLE
WebSocket API docs are incomplete

### DISCUSSION THREAD

**Discussion by artyomkonyaev** [deepgram/community#457] [General help] [answered]
I'm looking for an instruction on how to use Streaming API via plain WebSockets, without Deepgram's SDKs.

The only page I could find is this one:
https://developers.deepgram.com/reference/streaming

It doesn't give any idea whatsoever on how I can configure recognition parameters.

Where can I find this information?

I am using this url:
`wss://api.deepgram.com/v1/listen`
With this header:
`Authorization: Token {MY_API_TOKEN}`

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @artyomkonyaev, you may find our [Streaming Test Suite](https://github.com/deepgram/streaming-test-suite/tree/main) helpful. It is a resource meant for testing and debugging Deepgram streaming, and it is written in Python using websockets without any SDKs. [Here](https://github.com/deepgram/streaming-test-suite/blob/main/test_suite.py#L83-L152) is the specific section using the websocket.

Let us know if you're working in a programming language other than Python, and I'll see if we have resources more pertinent to your use case.
